tox.ini

```diff
@@ -8,7 +8,7 @@ install_command = pip install --use-wheel {opts} {packages}
 deps = -rrequirements-dev.txt
 commands =
     coverage erase
-    coverage run -m pytest {posargs:tests}
+    coverage run -m pytest --basetemp=tmp {posargs:tests}
```

Comment:
You can accomplish this in userspace by doing:

`TMPDIR=tmp tox -e py27` (or `TMPDIR=tmp make test`)